src/js/load-image.js

```diff
@@ -0,0 +1,17 @@
+const canvas = document.querySelector('.canvas-main');
+const ctx = canvas.getContext('2d');
+
+function loadImage() {
+  const imgDefault = localStorage.getItem('img');
+  const imgNode = new Image();
+  imgNode.src = imgDefault;
```

Comment:
Since `imgDefault` is not changing it is basically:
```suggestion
  imgNode.src = localStorage.getItem('img');
```